Kingfisher/ImageCache.swift

```diff
@@ -146,7 +146,7 @@ public extension ImageCache {
     - parameter key:          Key for the image.
     */
     public func storeImage(image: UIImage, originalData: NSData? = nil, forKey key: String) {
-        storeImage(image, originalData: originalData,forKey: key, toDisk: true, completionHandler: nil)
+        storeImage(image, originalData: originalData, forKey: key, toDisk: true, completionHandler: nil)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 104 characters